cellpack/autopack/writers/__init__.py

```diff
@@ -175,7 +175,8 @@ def save_as_simularium(self, env, all_ingr_as_array, compartments):
         file_name = env.helper.writeToFile(
             env.result_file, env.boundingBox, env.name, env.version
         )
-        autopack.helper.post_and_open_file(file_name)
+        if env.config_data is None or env.config_data.get("upload_results", True):
```

Comment:
This check is a bit confusing for me. We're uploading and posting results if the `config_data` is missing or if `upload_results` is not set?

I would consider adding another check on `number_of_packings` to make sure we don't end up opening multiple browser tabs if `upload_results` is not set or missing on a config with 100s of packings.
```suggestion
        if env.config_data is None or env.config_data.get("upload_results", env.config_data.get("number_of_packings", 1) <= 1):
```

Reply:
Thank you for your suggestion of adding a check for `number_of_packings`! If `number_of_packings` is always going to be specified in parallel packings, then we should be able to prevent opening multiple browser tabs by just checking the `number_of_packings` key.

That way, we wouldn't need to worry about adding the `upload_results` key to the config.

For example, we could use `if env.config_data.get("number_of_packings", 1) <= 1):`